meta-mender-imx/README.md

```diff
@@ -0,0 +1,58 @@
+# meta-mender-imx
+
+Mender integration for imx based boards
+
+The supported and tested boards are:
+
+ - [i.MX8M Nano EVK](TBD)
```

Comment:
It is this one right? https://hub.mender.io/t/nxp-i-mx-8m-mini-evaluation-kit/659

Reply:
No.  That's the mini.  I have not created the hub page for it yet.